app/views/users/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<section class='users-list'>
+  <% @users.each do |user| %>
+    <%= render partial: 'portials/user', locals: {user: user} %>
```

Comment:
Please make sure that clicking on each user takes the user to the user show page as shown in [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAAB2WHK6H4HSKPABF2UWG6T3IY3CYROA) wireframe. There is an error when a user visits the user show page.

![show_error](https://user-images.githubusercontent.com/35504149/199572342-4cda4232-3be7-490c-9ef9-a487d9c17f86.png)